modules/nf-core/merquryfk/merquryfk/main.nf

```diff
@@ -6,25 +6,27 @@ process MERQURYFK_MERQURYFK {
     container 'ghcr.io/nbisweden/fastk_genescopefk_merquryfk:1.2'
 
     input:
-    tuple val(meta), path(fastk_hist), path(fastk_ktab), path(assembly), path(haplotigs)
+    tuple val(meta), path(fastk_hist),path(fastk_ktab),path(asm1),path(haplotigs)
+    path matktab                                                                        //optional
+    path patktab                                                                        //optional
```

Comment:
Usage wise, would these normally be a paired tuple? Or does it make it problematic having two lists of files as a tuple?

Reply:
I tried list, but failed at testing.